gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/jvmkill.c

```diff
@@ -17,9 +17,18 @@
 #include <stdio.h>
 #include <string.h>
 #include <unistd.h>
-
+#include <stdlib.h>
 #include <jvmti.h>
 
+void logMessage(char * message){
+  FILE * logFile = fopen("jvmkill.log", "w");
+  if(logFile == NULL){
+    fprintf(stderr, "Error opening log file");
+  }
+  fputs(message, logFile);
```

Comment:
fputs and fclose should do in else portion. fprintf can print the entire message to stderr for the NULL case.